### PR TITLE
docs(angular): Add support for Angular version 15+

### PR DESCRIPTION
**Pull request by rifont**
**Summary**
* Update Angular client and quickstart docs to specify support for version 15+, per https://github.com/novuhq/novu/pull/4518
* Formatting fixes on Angular quickstart
  * Remove extra lines
  * Add code-blocks for install CLI commands
* Adds a snippet for supported channels with some visual tidy-up

**Screenshots**
_Angular Client_
image

_Supported Channels Snippet_
image

### CODE REVIEW

**Comment by rifont** [comment]
We'll need to hold off on merging this until we release support for Angular 15+ in v0.20, as part of https://github.com/novuhq/novu/pull/4518

File: _snippets/supported-channels.mdx
Comment (rifont) [inline _snippets/supported-channels.mdx:8]: This has 20+ occurrences across the docs, and is likely to change as we support more channels, so I've created a snippet.

File: notification-center/client/angular.mdx
Comment (rifont) [inline notification-center/client/angular.mdx:23]: `CodeGroup` for a nicer experience to install. Defaults to `pnpm`

File: notification-center/client/angular.mdx
Comment (rifont) [inline notification-center/client/angular.mdx:74]: No need to install `@types/react` anymore per https://github.com/novuhq/novu/pull/4518.

File: quickstarts/angular.mdx
Comment (rifont) [inline quickstarts/angular.mdx:89]: Had some non-standard space characters here previously.

File: quickstarts/angular.mdx
Comment (rifont) [inline quickstarts/angular.mdx:202]: As above, no longer required.

File: quickstarts/angular.mdx
Comment (rifont) [inline quickstarts/angular.mdx:193]: Simplifying this down into a diff, as some users may have a different looking `tsconfig` they are integrating with.

File: quickstarts/angular.mdx
Comment (rifont) [inline quickstarts/angular.mdx:209]: Out of the box alignment into the `.html` file.

**Review by rifont — COMMENTED** [review]
I have made a note on Linear to merge this PR when `v0.21.0` is released.

It is ready for review.

**Review by scopsy — APPROVED** [review]

**Pull request merged**
